### PR TITLE
10 ready for review too

### PR DESCRIPTION
**Pull request by LKFULLER1**

### CODE REVIEW

File: test [hunk 1]
```diff
@@ -0,0 +1,6 @@
+You are now connected to database "nc_games_test" as user "srvadmin".
+ review_id |       title       |     category     |   designer   |   owner    |          review_body           |                              review_img_url                               |       created_at        | votes | comment_count
```
Comment (MKD01) [inline test]: happy for this to be used but please do add this to your .gitignore as this does not need to be on github.

File: test.sql [hunk 2]
```diff
@@ -0,0 +1,8 @@
+\c nc_games_test
+SELECT reviews.*, 
+    CAST(COUNT(*)AS int) AS comment_count 
+    FROM reviews
```
Comment (MKD01) [inline test.sql]: same as above this does not need to be commit up to github

**Review by MKD01 — APPROVED** [review]
Happy for this to be merged once those changes have been made.

File: test.sql (hunk 2 above)
Comment (LKFULLER1) [reply test.sql]: Think I've deleted them now

**Pull request merged**